cafebabel/templates/articles/drafts/edit.html

```diff
@@ -1,49 +1,71 @@
 {% extends "base.html" %}
 
+{% block body_class %}create-article-page{% endblock %}
 {% block body %}
 <form method=post action={{ url_for('drafts.edit', draft_id=article.id) }} enctype=multipart/form-data>
   <input type=hidden name=id value="{{ article.id }}">
-  <div class=text>
+  <div class=file>
+      <label for=image>Image</label>
+      {% if article.has_image %}
+          <img src={{ article_image_url(article) }} alt>
+          <div class=checkbox>
+            <label for=delete-image>Delete image</label>
+            <input type=checkbox name=delete-image id=delete-image value=1>
+          </div>
+      {% endif %}
+      <input type=file id=image name=image>
+  </div>
+  <h1 class='edit active'>
```

Comment:
Please keep the double quote convention here too

Reply:
ok